pkgs/development/compilers/gcc-arm-embedded/11/default.nix

```diff
@@ -40,16 +40,16 @@ stdenv.mkDerivation rec {
     find $out -type f | while read f; do
       patchelf "$f" > /dev/null 2>&1 || continue
       patchelf --set-interpreter $(cat ${stdenv.cc}/nix-support/dynamic-linker) "$f" || true
-      patchelf --set-rpath ${lib.makeLibraryPath [ "$out" stdenv.cc.cc ncurses5 python38 libxcrypt-legacy ]} "$f" || true
+      patchelf --set-rpath ${lib.makeLibraryPath [ "$out" stdenv.cc.cc ncurses5 python39 libxcrypt-legacy ]} "$f" || true
     done
   '';
 
   postFixup = ''
     mv $out/bin/arm-none-eabi-gdb $out/bin/arm-none-eabi-gdb-unwrapped
     cat <<EOF > $out/bin/arm-none-eabi-gdb
     #!${runtimeShell}
-    export PYTHONPATH=${python38}/lib/python3.8
-    export PYTHONHOME=${python38}/bin/python3.8
+    export PYTHONPATH=${python39}/lib/python3.9
+    export PYTHONHOME=${python39}/bin/python3.9
```

Comment:
We have python.interpreter for this

Reply:
That's not what is supposed to be done here

gcc-arm-embedded have binaries that require python38 explicitly, if we don't have it, we should just mark the package as broken

Reply:
> because it is a binary and not built from source

I don't understand this part. We only provide source-built Python in nixpkgs.

Reply:
gcc-arm-embedded is not built from source

but nevermind - let's get this PR in and I will address the issue in another PR

Reply:
I'm not in a hurry to get it in, I just wanted to get attention to the issue early.